e2e/vue3-vite-e2e/tests/docs.spec.ts

```diff
@@ -28,7 +28,7 @@ describe('docs e2e', () => {
 
     // Build
     const result = await runNxCommandAsync(`build ${docs}`);
-    expect(result.stdout).toContain('Build complete');
+    expect(result.stdout).toContain('build complete');
```

Comment:
That's strange, thanks for the fix! I should get around to updating the vitepress stuff, but since I'm not currently working on a vitepress project, it hasn't been high priority.